### PR TITLE
More logs

### PR DESCRIPTION
**Pull request by ebrossier-secher**
Add initialisation logs for the bot and make previous ones a bit more detailed.

### CODE REVIEW

File: blogReader.py [hunk 1]
```diff
@@ -15,8 +15,10 @@ def __init__(self) -> None:
         self.__cache = []
         self.__new = []
         self.logger = logging.getLogger(__name__)
+        self.logger.info("Bot initialisation.")
         self.__getArticles()
         self.__cacheArticlesID()
+        self.logger.info("'cacheArticlesID' initialisated")
```
Comment (ctmbl) [inline blogReader.py]:
```suggestion
        self.logger.info("'cacheArticlesID' initialized")
```
or better
```suggestion
        self.logger.info("articles cache initialized")
```

File: blogReader.py [hunk 2]
```diff
@@ -29,13 +31,13 @@ def __cacheArticlesID(self):
...
         for dic in self.__cache:
             self.__cachedArticlesID.append(dic['_id'])
-        self.logger.info("'cacheArticlesID' updated")
 
     def recache(self):
         new_articles = self.checkUpdate()
         self.__cache = []
         self.__getArticles()
         self.__cacheArticlesID()
+        self.logger.info("'cacheArticlesID' updated")
```
Comment (ctmbl) [inline blogReader.py]: I'd prefer 
```suggestion
        self.logger.info("articles cache updated")
```
becausde the __cache array ios updated in factr, not the method you know

File: main.py [hunk 3]
```diff
@@ -42,9 +42,10 @@ async def auto_send():
 
 if __name__ == "__main__":
     try:
-        bot.run(BOT_TOKEN, log_handler=None) # Setting log_handler to None allows
-                                             # to propagate the handler to root for processing.
+        # Setting log_handler to None allows to propagate the handler to root for processing.
```
Comment (ctmbl) [inline main.py]:
```suggestion
        # Setting log_handler to None allows to propagate the logs to the root logger for processing.
```

**Review by ctmbl — CHANGES_REQUESTED** [review]
minor changes in formaulation but tks for the update!

**Comment by ebrossier-secher** [comment]
> minor changes in formaulation but tks for the update!

It should be all good now !

**Review by ctmbl — APPROVED** [review]

**Pull request merged**